websites/A/Animeflix/presence.ts

```diff
@@ -0,0 +1,29 @@
+const presence = new Presence({
+		clientId: "1039178085922250873",
+	}),
+	strings = presence.getStrings({
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused",
+	});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+    largeImageKey: "https://i.imgur.com/4hiyxuW.png"
+  }, { pathname } = document.location;
+
+  console.log(pathname)
+  if (pathname === "/") presenceData.details = "Exploring Main Page";
+  else if(pathname === "/genres") presenceData.details = "Exploring Genres";
```

Comment:
You can add a state. For example, when the anime information window appears, display the title. This also applies to the following lines.